omnibus/files/server-ctl-cookbooks/infra-server/attributes/default.rb

```diff
@@ -78,11 +81,11 @@
 # The Chef User that services run as
 ####
 # The username for the chef services user
-default['private_chef']['user']['username'] = 'opscode'
+default['private_chef']['user']['username'] = ChefUtils::Dist::Org::LEGACY_CONF_DIR
```

Comment:
this is a username, but we're using a dirname const. If we want to change the default username, this should have its own const.

Reply:
@marcparadise Would you like us to add something like `ChefUtils::Dist::Server::USERNAME` to chef-utils?

Reply:
@marcparadise if this works for you - https://github.com/chef/chef/pull/12513/files